### PR TITLE
Seed sectors (names and CP benchmark units)

### PR DESCRIPTION
**Pull request by kowal**
Units are taken from http://www.lse.ac.uk/GranthamInstitute/tpi/the-toolkit/

Zrzut ekranu 2019-09-17 o 08 32 11
Zrzut ekranu 2019-09-17 o 08 32 24

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1 +1,26 @@
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
+if Rails.env.development?
```
Comment (simaob) [inline db/seeds.rb]: Should we move the Units to outside the development env? As we'll need them in prod and staging as well.

File: app/admin/sectors.rb [hunk 2]
```diff
@@ -58,6 +60,7 @@
 
     f.inputs do
       f.input :name
+      f.input :cp_unit, label: 'Carbon Performance benchmark unit'
```
Comment (tsubik) [inline app/admin/sectors.rb]: should we just call it `Carbon Performance Unit` ?

Comment (kowal) [reply app/admin/sectors.rb]: year, works for me :) @simaob ?

File: db/seeds.rb (hunk 1 above)
Comment (kowal) [reply db/seeds.rb]: Right! Also, since we can edit them in the interface I'll add a condition to only set units from seeds for new records.

**Pull request merged**